app/controllers/application_controller.rb

```diff
@@ -84,7 +84,7 @@ def request_event
 
   def trigger_feedback_provided_event
```

Comment:
I reckon this might be better off in a concern that is included in the various feedback controllers rather than in `ApplicationController`